db/migrate/20150827123826_remove_scaptimony_permissions.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveScaptimonyPermissions < ActiveRecord::Migration
+  def up
+    permissions = Permission.where(:resource_type => ["Scaptimony::Policy", "Scaptimony::ScapContent"])
+    permissions.each do |p|
+      p.resource_type = p.resource_type.sub(/^Scaptimony/, "ForemanOpenscap")
```

Comment:
could you add more info to this, if there is ever an error while saving it would be hard to debug, so something like 
```say "Converting permission #{p.id} with name '#{p.name}' of type #{p.resource_type}"```
would be helpful